clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListCatalogs.java

```diff
@@ -35,29 +36,26 @@ public class ListCatalogs extends Command {
    * @param url The URL of the Gravitino server.
    * @param ignoreVersions If true don't check the client/server versions match.
    * @param metalake The name of the metalake.
+   * @param listOptions The list options.
    */
-  public ListCatalogs(String url, boolean ignoreVersions, String metalake) {
-    super(url, ignoreVersions);
+  public ListCatalogs(
+      String url, boolean ignoreVersions, String metalake, ListOptions listOptions) {
+    super(url, ignoreVersions, listOptions);
     this.metalake = metalake;
   }
 
   /** Lists all catalogs in a metalake. */
   @Override
   public void handle() {
-    String[] catalogs = new String[0];
+    Catalog[] catalogs;
     try {
       GravitinoClient client = buildClient(metalake);
-      catalogs = client.listCatalogs();
+      catalogs = client.listCatalogsInfo();
+      output(catalogs);
```

Comment:
This command is a list command that only lists the catalog names. The details command lists more information. This is now inconsistent with other commands. Why was this changed?

Reply:
OK I understand